### DISCUSSION TITLE
Change ALL the stylesheets at once

### DISCUSSION THREAD

**Discussion by barbas66** [JaKooLit/Hyprland-Dots#337] [Q&A]
Hi there,

I'd like to modify the behavior of taskbar module for all the stylesheets.

If I import an ad hoc stylesheet (i.e. _taskbar.css_) at the beginning of each stylesheets (e.g. _@import "../waybar/style/taskbar.css_), will the settings of the imported file supersede the settings already present in the stylesheet or will the latter survive and take precedence?

Thanks for the help!

**Comment by JaKooLit** [comment]
never tried that tbh

but in theory it should work.

but you still need to edit individual css styles to add that @import..
